AY2021S2-CS2103-T14-1/tp
# Add command history storage

## What this does
This PR
1. adds save/load capabilities to command history
2. updates the UG with information about the saved command history

Closes #97 

## How to test
1. Launch SunRez.
2. Perform some normal commands.
3. Exit and re-launch SunRez.
4. Execute the `history` command. You should see the history from the last session listed.
5. Press the UP and DOWN arrow keys. You should see SunRez navigating through your command history, including entries from the previous session.

## Notes
- Command history is saved as a plain-text file at `[JAR file location]/data/commandhistory.txt`.
- Each command history entry is a line within the file.
